app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express")
+const app = express()
+const { getTopics } = require("./controllers/news_controller")
+
+app.get("/api/topics", getTopics)
+
+app.use((err, request, response, next) => {
+    if (err.code) console.log(err);
+    else next(err);
+});
+
+
+app.use((err, request, response, next) => {
+    console.log(err);
+    response.status(500).send("Internal server error");
+});
+
```

Comment:
I like this error handling setup so far, but you could also add a generic 404 here with the appropriate testing as well